pkg/ds/farm.go

```diff
@@ -564,6 +565,25 @@ func (dsf *Farm) spawnDaemonSet(
 
 	desiresCh := ds.WatchDesires(quitSpawnCh, updatedCh, deletedCh)
 
+	go func() {
+		aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
+		ticks := time.NewTicker(1 * time.Minute)
+
+		defer aggregateHealth.Stop()
```

Comment:
After a replication ends, there is nobody reading off of the aggregator's channel anymore, so it blocks itself sending on the channel. Therefore, it no longer does any more watches. Therefore, the only impact is we accumulate goroutines that are blocked on channels, but we don't kill the network/consul, which is at least not as bad as I thought it would be.